src/main/kotlin/lotto/domain/Tickets.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+class Tickets(val tickets: List<Ticket>) {
+    constructor(amountOfTicket: Int) : this(generateTickets(amountOfTicket))
+
+    companion object {
+        private fun generateTickets(amountOfTicket: Int): List<Ticket> {
+            return List(amountOfTicket) { Ticket() }
+        }
+    }
+
+    fun size() : Int {
+        return tickets.size
+    }
```

Comment:
```suggestion
    val size: Int
       get() = tickets.size
```

You could use a `property` instead of a function here.
What do you think are the benefits of using a property in this case?

Here’s a reference you might find helpful:
- [[Kotlin Docs] Properties](https://kotlinlang.org/docs/properties.html)

Reply:
Oh, I see. I guess this example is about a property in Kotlin, because `size()` is not really about behavior but about getting a state.
There might not be a huge difference between a property and a function, but using size as a property can make the code more readable.